### ISSUE TITLE
Coordinate with hera_sim to generate RFI for validation

### ISSUE THREAD

**Issue by adampbeardsley** [HERA-Team/hera_qm#270]
@steven-murray let's chat about this.

**Comment by adampbeardsley** [comment]
Does hera_sim have capability to make "DTV-like" events?

**Comment by steven-murray** [comment]
What does a "DTV-like" event look like?

**Comment by adampbeardsley** [comment]
@mwilensky768 can probably give a more quantitative answer, but my understanding is that DTV has broader bandwidth than a single channel (~10s of channels?). I've also seen it more like impulses rather than continuously on. But maybe that's specific to things that are reflecting?

**Comment by bhazelton** [comment]
Digital TV (DTV) signals have very well-defined shapes (because they're part of spectrum allocations) that vary a bit country-to-country, @mwilensky768  has looked up the specifics for South Africa. We have studied them more with the MWA, where we primarily see them as reflections off of airplanes that last 10s of seconds (Mike knows more on this too). We've seen them in HERA data as well, and they look fairly similar to MWA events, but we haven't studied them enough to be sure that they are from aircraft reflections.

**Comment by mwilensky768** [comment]
This plan shows that television broadcasting exists in South Africa between 174 and 214 Mhz (and other frequencies too, but these are the relevant ones):

https://www.icasa.org.za/uploads/files/National-Radio-Frequency-Plan-2018-41650.pdf

Page 127 of this document shows the spectrum characteristics of the broadcasting system that they use in South Africa (a PSD plot):

https://www.etsi.org/deliver/etsi_en/302700_302799/302755/01.04.01_60/en_302755v010401p.pdf

In summary, they are 8 Mhz wide (actually like 7.875 with some guard bands and some dropoffs - detailed in the plot in the above link) and broadcast immediately adjacent to one another starting at 174 Mhz.

**Comment by steven-murray** [comment]
Thanks @bhazelton and @mwilensky768 . So short answer is no, hera_sim can't do that. Slightly longer answer is that it sounds reasonably easy, so maybe I can do it today.

**Comment by adampbeardsley** [comment]
Another point that came up on the telecon yesterday was time dependence of noise that @mwilensky768 sees in his SSINS analysis. It's not exactly RFI related, but it affects how SSINS detects RFI and may affect XRFI as well. Might be worth putting this functionality in.
Also, as I write this, it strikes me that HERA might have a stronger time dependent sky temperature than MWA just because of the smaller field of view. We should look into whether that accounts for it.

**Comment by mkolopanis** [comment]
This is actually coming along well. @adampbeardsley  @steven-murray  have been working on this and have a notebook running that is attempting to make this work. Any other updates you two want to give?

**Comment by steven-murray** [comment]
not quite yet. The notebook is coming along -- there's still a few things that definitely need to be done. Perhaps we'll upload the notebook to somewhere soon so it can be critiqued more broadly.
